cookiecutter.json

```diff
@@ -23,6 +23,6 @@
     "_copy_without_render": [
         "docs/_templates"
     ],
-    "minimum_python_version": ["2.7", "3.5", "3.6"],
+    "minimum_python_version": ["3.5", "3.6", "2.7"],
```

Comment:
Nitpick: Why this order? Why not just straight reverse numerological (sp?) order (e.g., 3.6, 3.5, 2.7)?

Reply:
The first one is the default, I thought it was better to default to 3.5 not 3.6, although I am happy to default to 3.6

Reply:
In other projects, I have seen performance improvement in 3.6 compared to 3.5 IMHO 3.6 should be default. Is there a reason why it must be 3.5? @bsipocz ?

Reply:
Either way is fine by me, but slightly prefer @pllim's suggestion